triggerbot/tree_watcher.py

```diff
@@ -57,8 +58,10 @@ def __init__(self, ldap_auth, is_triggerbot_user=lambda _: True):
         self.auth = ldap_auth
         self.lower_trigger_limit = TreeWatcher.default_retry * TreeWatcher.per_push_failures
         self.log = logging.getLogger('trigger-bot')
+        logging.basicConfig()
         self.is_triggerbot_user = is_triggerbot_user
         self.global_trigger_count = 0
+        self.treeherderapi = TreeherderApi()
```

Comment:
With this and #4 we'll have a treeherderapi and threeherder_client in this class. Can you find a name for this that's has more to do with the data we get from it than the place we're getting it from?